Source/cmake/WebKitFeatures.cmake

```diff
@@ -191,6 +191,7 @@ macro(WEBKIT_OPTION_BEGIN)
     WEBKIT_OPTION_DEFINE(ENABLE_LAYOUT_TESTS "Toggle layout test support (DumpRenderTree/WebkitTestRunner)" PRIVATE OFF)
     WEBKIT_OPTION_DEFINE(ENABLE_LETTERPRESS "Toggle letterpress support" PRIVATE OFF)
     WEBKIT_OPTION_DEFINE(ENABLE_MANAGED_MEDIA_SOURCE "Toggle Managed Media Source support" PRIVATE ON)
+    WEBKIT_OPTION_DEFINE(ENABLE_LOGS "Toggle logging support" PRIVATE OFF)
```

Comment:
This should be alphabetized (one line higher).

But also, if it's going to be a cross-platform option, the implementation should go in some cross-platform place. At the very least, you could put it in OptionsCommon.cmake rather than OptionsGTK/WPE.cmake. But it should probably be done in PlatformEnable.h.